flask_saml_sso/sso.py

```diff
@@ -244,13 +315,24 @@ def sls(auth):
     Consumes LogoutResponse from IdP when logout has been performed, and
     sends user back to landing page
     """
+    logger = _get_logger()
+    logger.debug('SLS called')
+
     url = auth.process_slo(delete_session_cb=lambda: flask.session.clear())
+    logger.debug('SLS Response XML: \n{}'.format(
+        auth.get_last_response_xml()))
+
     errors = auth.get_errors()
     if errors:
+        logger.error('Errors: {}'.format(errors))
```

Comment:
What are we doing here? A sudden “Errors:” in the log might not be terribly helpful…

Also; you're invoking `process_slo` in `sls`; is that intentional?

Reply:
Yes.

Reply:
> Yes.

That's vaguely suprising — a comment would be awfully nice :)